freestanding/Makefile

```
@@ -0,0 +1,23 @@
+PKG_CONFIG_PATH := $(shell opam config var prefix)/lib/pkgconfig
+
+EXISTS := $(shell PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) pkg-config --exists ocaml-freestanding; echo $$?)
+
+.PHONY: all clean
+all: libdigestif_freestanding_stubs.a
+
+ifeq ($(EXISTS), 1)
+libdigestif_freestanding_stubs.a:
+	touch $@
+else
+CC ?= cc
+FREESTANDING_CFLAGS := $(shell PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) pkg-config --cflags ocaml-freestanding)
+CFLAGS := -I. $(FREESTANDING_CFLAGS)
```

```suggestion
CFLAGS := -O3 $(FREESTANDING_CFLAGS)
```

add optimization level, and I don't think the `-I.` is needed (it is the default to have the current working directory being part of the include search path).